pkg/dataplane/test/sync_test.go

```diff
@@ -1367,22 +1367,26 @@ func (suite *syncStreamBackupRestoreTestSuite) TestStream() {
 	for _, cursorItem := range cursorItems {
 		shardName, err := cursorItem.GetFieldString("__name")
 		suite.Require().NoError(err, "Failed to get item name")
-		chunkId, chunkDataArray, chunkMetadata, currentChunkMetadata, err := cursorItem.GetShard()
+		chunkMap, currentChunkMetadata, err := cursorItem.GetShard()
 		suite.Require().NoError(err, "Failed to get stream")
 
 		if _, ok := streamBackup[shardName]; !ok {
 			streamBackup[shardName] = &Shard{Chunks: map[int]*Chunk{}}
 		}
 
-		if _, ok := streamBackup[shardName].Chunks[chunkId]; !ok {
-			streamBackup[shardName].Chunks[chunkId] = &Chunk{}
-		}
-		if chunkMetadata != nil {
-			(*streamBackup[shardName].Chunks[chunkId]).ChunkMetadata = chunkMetadata
-		}
-		if len(chunkDataArray) != 0 {
-			(*streamBackup[shardName].Chunks[chunkId]).Data = chunkDataArray
+		for chunkId := range chunkMap {
+			if _, ok := streamBackup[shardName].Chunks[chunkId]; !ok {
+				streamBackup[shardName].Chunks[chunkId] = &Chunk{}
```

Comment:
`chunkMap` is already map of chunks by `chunkID`

Reply:
you right, but there could be a scenario that there is chunks that is scattered over multiple Items...

Reply:
what I meant is - you have two places where you define the same `Chunk` struct

Reply:
point me to which ones?

Reply:
https://github.com/v3io/v3io-go/blob/9e9b44868989d454367ad3e0c6c6cf8e55009e24/pkg/dataplane/types.go#L473 
and 
https://github.com/v3io/v3io-go/blob/9e9b44868989d454367ad3e0c6c6cf8e55009e24/pkg/dataplane/test/sync_test.go#L1358

Reply:
you are right, change to the same struct